ql/src/java/org/apache/hadoop/hive/ql/Compiler.java

```diff
@@ -188,7 +188,6 @@ private BaseSemanticAnalyzer analyze() throws Exception {
     // because at that point we need access to the objects.
     Hive.get().getMSC().flushCache();
 
-    driverContext.setBackupContext(new Context(context));
```

Comment:
Are we sure about this?
We create a backup context so if we have to reexecute the query then we have a context at hand (for removing temporary files etc)

Reply:
The usage of backupContext was removed by Peter Varga recently (https://github.com/apache/hive/commit/e2a02f1b43cba657d4d1c16ead091072be5fe834#diff-71a166c053d9c698f9cb64eaef832aff), I've asked him to confirm, and it was intentional. After this change we are only setting the backup context here, but it is never used.